docs/reference/setup/bootstrap-checks.asciidoc

```diff
@@ -23,39 +23,43 @@ documented individually.
 [float]
 === Development vs. production mode
 
-By default, Elasticsearch binds to `localhost` for <<modules-http,HTTP>>
-and <<modules-transport,transport (internal)>> communication. This is
-fine for downloading and playing with Elasticsearch, and everyday
-development but it's useless for production systems. To form a cluster,
-Elasticsearch instances must be reachable via transport communication so
-they must bind transport to an external interface. Thus, we consider an
-Elasticsearch instance to be in development mode if it does not bind
-transport to an external interface (the default), and is otherwise in
-production mode if it does bind transport to an external interface.
+By default, Elasticsearch binds to `localhost` for <<modules-http,HTTP>> and
+<<modules-transport,transport (internal)>> communication. This is fine for
+downloading and playing with Elasticsearch, and everyday development but it's
+useless for production systems. To join a cluster, an Elasticsearch node must be
+reachable via transport communication; to join a cluster over an external
+network interface, a node must bind transport to an external interface and not
+be using <<single-node-discovery,single-node discovery>>. Thus, we consider an
+Elasticsearch node to be in development mode if it can not form a cluster with
+another machine over an external network interface, and is otherwise in
+production mode if it can join a cluster over an external interface.
 
 Note that HTTP can be configured independently of transport via
```

Comment:
I think this should be

"Note that HTTP and transport communication can be configured independently via ..."

Reply:
Thanks.